helpers-json/utils.js

```diff
@@ -92,7 +92,7 @@ function flattenAndSplitResources(keys, obj) {
  *   }
  * }
  * const ph = arbPlaceholderHandler(placeholders)
- * assert(ph === "PH(count|1|number of tickets")
+ * assert(ph === "PH({{count}}|1|number of tickets")
```

Comment:
Nit: there should be a ")" after tickets

Reply:
done